lib/codec.js

```diff
@@ -18,6 +18,7 @@
   exports.encodeBytes = f.lift(doEncodeBytes, _.identity);
   exports.encodeString = f.lift(doEncodeString, _.identity);
   exports.encodeSignedInt = f.lift(doEncodeSignedInt, _.identity);
+  exports.encodeShort = f.lift(doEncodeShort, _.identity);
```

Comment:
To provide a bit more information on this, it appears that the root cause in [JDG-94](https://issues.jboss.org/browse/JDG-94) is how JBoss Strings are encoded when adding the script. I have the feeling is that if the script is longer than certain size it fails with the error in the issue. So, what's relevant here to test is how a particular String of certain size is converted into a JBoss Marshalling-encoded String. This is more relevant than simply testing how shorts are encoded on their own. In fact, by testing Strings of different sizes you indirectly test the short encoding and encoding JBoss Strings is something already exposed to the rest of components in the project, so no need to overexpose anything.